.github/workflows/ci.yml

```diff
@@ -3,39 +3,53 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 10.x
-    - name: Install Dependencies
-      run: npm install
-    - name: Lint
-      run: npm run lint
-
-  test:
-
+  tests:
     runs-on: ${{ matrix.os }}
 
+    timeout-minutes: 20
+
     strategy:
-      fail-fast: false
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
-        node-version: [10.x]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        node-version: [12.x, 14.x]
+
+    env:
+      HEADLESS: true
 
     steps:
+    - name: Prepare git
+      run: git config --global core.autocrlf false
+
     - uses: actions/checkout@v2
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: Install Dependencies
+
+    - name: Get npm cache directory
+      id: npm-cache
+      run: |
```

Comment:
As there's one command to run, can the line for `|` be removed? Is it OK to have this line as:

```
run: echo "::set-output name=dir::$(npm config get cache)"
```

The same comment applies to line 39.